### PR TITLE
Agreements and intervention unsuspend 

### PR DESCRIPTION
**Pull request by daniel-tabarcea**
- unsuspend working
- fixed icon issues on agreements page
- connects unicef/etools-issues#508

### CODE REVIEW

File: pmp/app-elements/agreements/agreement-status.html [hunk 1]
```diff
@@ -333,7 +333,7 @@
           }
           if (newStatus === 'suspended' || newStatus === 'terminated') {
             if (this.newAgreement || (this.status !== 'suspended'
-              && this.status !== 'terminated' && this.status !== 'active')) {
+              && this.status !== 'terminated' && this.status !== 'signed')) {
               return false;
```
Comment (acory) [inline pmp/app-elements/agreements/agreement-status.html]: can you add a comment for these ifs ? it's hard to follow what's trying to be avoided here

File: pmp/app-elements/agreements/agreement-status.html [hunk 2]
```diff
@@ -350,7 +350,7 @@
         },
 
         _setStatusNotSuspended: function() {
```
Comment (acory) [inline pmp/app-elements/agreements/agreement-status.html]: _setStatusNotSuspended sounds a bit off , _unsuspendAgrement may be better?

File: pmp/app-elements/agreements/agreement-status.html (hunk 1 above)
Comment (daniel-tabarcea) [inline pmp/app-elements/agreements/agreement-status.html]: I won't add a comment, but i'll try to rewrite it so it's more readable
